src/theme/components.js

```diff
@@ -55,29 +55,9 @@ const components = {
       },
     },
   },
-  MuiTable: {
```

Comment:
all the styling removed was in place to create a compact UI, now handled by `dense` and `size` props